### PR TITLE
fix: Increase max gRPC response msg size

### PR DESCRIPTION
**Pull request by pvaneck**
Some responses (e.g. from BERT models) are larger than the default of 4194304 bytes. This commit updates the max message size to be 16Mib to align with MM and runtime defaults. This value is also configurable through env variables.

Minor refactoring done to handle all the integer env variables.

### CODE REVIEW

**Comment by pvaneck** [comment]
FYI @tedhtchang

File: proxy/main.go [hunk 1]
```diff
@@ -34,19 +34,35 @@ import (
...
 var (
-	grpcServerEndpoint   = "localhost"
+	grpcServerEndpoint = "localhost"
+	logger             = zap.New()
+
+	// Defaults
 	inferenceServicePort = 8033
-	logger               = zap.New()
 	listenPort           = 8008
+	maxGrpcSize          = 16777216
```
Comment (njhill) [inline proxy/main.go:51]: nit: I always prefer to include units.. e.g. `maxGrpcMessageSizeBytes`

File: proxy/main.go [hunk 2]
```diff
@@ -34,19 +34,35 @@ import (
 
 const (
 	restProxyPortEnvVar     = "REST_PROXY_LISTEN_PORT"
+	restProxyGrpcMaxMsgSize = "REST_PROXY_GRPC_MAX_MSG_SIZE"
```
Comment (njhill) [inline proxy/main.go:37]: nit: Would prefer `REST_PROXY_GRPC_MAX_MSG_SIZE_BYTES`

**Review by njhill — COMMENTED** [review]
Thanks @pvaneck, just a couple of nits

File: proxy/main.go (hunk 1 above)
Comment (pvaneck) [reply proxy/main.go]: Yea, that's sensible. Changed!

**Review by njhill — APPROVED** [review]

File: proxy/main.go (hunk 2 above)
Comment (pvaneck) [reply proxy/main.go]: Updated

**Pull request merged**